src/riak_cs_utils.erl

```diff
@@ -463,17 +462,11 @@ riak_connection(Pool) ->
 -spec set_object_acl(binary(), binary(), lfs_manifest(), acl(), riak_client()) ->
                             ok | {error, term()}.
 set_object_acl(Bucket, Key, Manifest, Acl, RcPid) ->
-    StartTime = os:timestamp(),
     {ok, ManiPid} = riak_cs_manifest_fsm:start_link(Bucket, Key, RcPid),
     _ActiveMfst = riak_cs_manifest_fsm:get_active_manifest(ManiPid),
     UpdManifest = Manifest?MANIFEST{acl=Acl},
     Res = riak_cs_manifest_fsm:update_manifest_with_confirmation(ManiPid, UpdManifest),
     riak_cs_manifest_fsm:stop(ManiPid),
-    if Res == ok ->
-            ok = riak_cs_stats:update_with_start([object, put_acl], StartTime);
-       true ->
-            ok
-    end,
     Res.
```

Comment:
What is the concern to use try block? The manifest fsm is linked from this process, doesn't it suffice?